alia-peterson/fitlit
# Feature/dashboard styling

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
Add chart for stairs climbed stats. 
Complete step challenge extension.



## *Instructions for reviewer:*
*Where should the reviewer start?*
scripts.js, chart.js
*How should this be manually tested?*
Cycle through site to see added functionality. 

*What are the relevant tickets?*
Close #34, close #35, close #36


## *Issues:*
Still need to work on doughnut chart styling.



## Checklist:
